backend/shared/types.d.ts

```diff
@@ -33,3 +33,8 @@ export interface GroupItem {
   visible: boolean;
   connections: GroupConnections;
 }
+
+export interface AWSError extends Error {
```

Comment:
This type no longer exists in the SDK?

Reply:
Apparently. In the DynamoDB typings, the callback for `createTable` has the error typed as `any`, same for the error in `postToConnection`. To avoid the anys, I created our own type.